### ISSUE TITLE
Create a server

### ISSUE THREAD

**Issue by duyminh1998** [Panopticon-AI-team/panopticon#15]
### We should
create a server that will run the simulation logic

### Because
running the game simulation in the frontend is causing us some bugs

### Complexity
Medium to hard

### Definition of Done
Frontend is only concerned with displaying data while a server can run the simulation

**Comment by orgulous** [comment]
Isn't this done?

**Comment by duyminh1998** [comment]
No. We do not have a server. We have a client-only web application and a Gymnasium environment.
